src/aidemo.py

```diff
@@ -21,10 +21,10 @@
 PIC_SIZE = {"hdmi": (300,300), "lvds": (225,225)}
 TRIGGER_BTN_SPACING = {"hdmi": 300, "lvds": 100}
 
-from ai import Ai
-from loadscreen import LoadScreen
-from camera import *
-
+from demo_celebrity_face_match.ai import Ai
+from demo_celebrity_face_match.loadscreen import LoadScreen
+from demo_celebrity_face_match.camera import *
+from demo_celebrity_face_match.config import *
```

Comment:
I am not a fan of wildcard imports, actually produces W0401 warnings. Especially, the PKG_DATA_DIR from the generated 'config' module produces an 'undefined variable' error in linter.